packages/components/src/components/drawer/style.scss

```diff
@@ -21,27 +22,51 @@
 				left: 0;
 				height: 100vh;
 				max-height: 100%;
+
+				#{$root}__content {
+					height: 100%;
+					max-width: rem(400);
```

Comment:
Anmerkung zur Breite und Höhe im Drawer:

Die Breite des Drawers sollte nicht fest definiert werden, sondern sich aus dem Inhalt ergeben. Das bedeutet, dass der Inhalt selbst eine feste Breite mitbringen muss, um das Layout zu bestimmen.

Für die Samples gilt:

Vertikale Drawer (z. B. von links/rechts): Die Breite wird vom Inhalt bestimmt.

Horizontale Drawer (z. B. von oben/unten): Die Höhe wird vom Inhalt bestimmt.

Bitte in den Beispielen darauf achten, dass die Inhalte (z. B. ein Container oder ein Card) eine feste Dimension (Breite oder Höhe) mitbringen, damit sich der Drawer korrekt verhält.